grammars/asciidoc.cson

```diff
@@ -30,7 +30,7 @@
   #
   #   === Level 2 Section
   {
-    'begin': '^={1,6}\\s+'
+    'begin': '^={1,6}\\p{Blank}+'
```

Comment:
I agree, hence \p{Blank}. Let's say you had a line `== Title\t` in your file. The original regex wouldn't match, whereas the \p{Blank} would.